pom.xml

```diff
@@ -52,6 +56,16 @@
 			<artifactId>jpa</artifactId>
 			<version>${spring-filter.version}</version>
 		</dependency>
+		<dependency>
```

Comment:
Use same version convention as for other dependencies (for example `<version>${spring-filter.version}</version>`)? Comment is valid for both dependencies below.